src/components/Modal/VotingPowerDelegationModal/VotingPowerDelegationModal.tsx

```diff
@@ -1,81 +1,60 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 
-import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
-import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
-import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
-import { Button } from 'decentraland-ui/dist/components/Button/Button'
 import { Close } from 'decentraland-ui/dist/components/Close/Close'
-import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import { Modal } from 'decentraland-ui/dist/components/Modal/Modal'
 
-import { CANDIDATE_ADDRESSES } from '../../../constants'
-import { EDIT_DELEGATE_SNAPSHOT_URL } from '../../../entities/Proposal/utils'
-import useDelegatesInfo, { Delegate } from '../../../hooks/useDelegatesInfo'
-import useVotingPowerDistribution from '../../../hooks/useVotingPowerDistribution'
-import DelegatesTable from '../../Table/DelegatesTable'
+import { VpDistribution } from '../../../clients/SnapshotGraphqlTypes'
+import VotingPowerDelegationDetail from '../VotingPowerDelegationDetail/VotingPowerDelegationDetail'
 
-import './VotingPowerDelegationModal.css'
+import VotingPowerDelegationCandidatesList, { Candidate } from './VotingPowerDelegationCandidatesList'
 
-type VotingPowerDelegationModalProps = Omit<ModalProps, 'children'> & {
-  setSelectedCandidate: (candidate: Candidate) => void
-  showPickOtherDelegateButton?: boolean
+type Props = {
+  vpDistribution: VpDistribution
+  openDelegationModal: boolean
+  setOpenDelegationModal: React.Dispatch<any>
+  displayedCandidate?: Candidate | null
 }
 
-export type Candidate = Delegate & {
-  bio: string
-  links: string[]
-  relevant_skills: string[]
-  involvement: string
-  motivation: string
-  vision: string
-  most_important_issue: string
-}
+const VotingPowerDelegationModal = ({
+  vpDistribution,
+  openDelegationModal,
+  setOpenDelegationModal,
+  displayedCandidate,
+}: Props) => {
+  const [selectedCandidate, setSelectedCandidate] = useState<Candidate | null>(null)
```

Comment:
I'd let the parent component handle what to do with the displayedCandidate.
`VotingPowerDelegationModal` can receive an `onClose` function to call here.